modules/extensions-support/src/test/java/org/mule/module/extension/HeisenbergOperations.java

```diff
@@ -151,4 +157,17 @@ public String callSaul(@Connection HeisenbergConnection connection)
     {
         return connection.callSaul();
     }
+
+    @Operation
+    public String callGusFring() throws HeisenbergException
+    {
+        throw new HeisenbergException("You are not allowed to speak with gus.");
```

Comment:
this is the exact same string you check for in the test case. Extract to constant